drivers/hmis/app/graphql/types/hmis_schema/query_type.rb

```diff
@@ -446,6 +446,33 @@ def project_configs
       Hmis::ProjectConfig.all
     end
 
+    field :can_project_accept_referral, Boolean, null: false do
+      argument :project_id, ID, required: true
+      argument :referrer_enrollment_id, ID, required: true
+    end
+    def can_project_accept_referral(project_id:, referrer_enrollment_id:)
```

Comment:
I'd also be grateful for feedback about the query name -- I think `can_project_accept_referral` is nice because it's simple, but it might be a bit confusing because there are other conditions for accepting referrals which we don't check here, like:
- project has CoC (?)
- project has available units